src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -13,66 +13,107 @@
 
     String allianceColor = DriverStation.getAlliance().toString();
     SmartDashboard.putString("Alliance Color", allianceColor);
   }
 
-  public void CDSBeltWheelControllerToggle(boolean reverse) {
+  public void CDSWheelToggle(boolean reverse) {
     if (reverse) {
+      CDSWheelControllerOne.getSparkMax().set(-Constants.CDSWheelControllerSpeed);
+      SmartDashboard.putString("CDS Wheel Direction", "Reverse");
+      SmartDashboard.putNumber("CDS Wheel Speed", -Constants.CDSWheelControllerSpeed);
+  
+    } else {
       CDSWheelControllerOne.getSparkMax().set(Constants.CDSWheelControllerSpeed);
-      SmartDashboard.putString("CDS Wheel Controller Direction", "Reverse");
-      SmartDashboard.putNumber("CDS Wheel Controller Speed", -Constants.CDSWheelControllerSpeed);
+      SmartDashboard.putString("CDS Wheel Direction", "Forward");
+      SmartDashboard.putNumber("CDS Wheel Speed", Constants.CDSWheelControllerSpeed);
       
+    }
+  }
+
+  public void CDSBeltToggle(boolean reverse) {
+    if(reverse){
       CDSBeltController.getSparkMax().set(-Constants.CDSBeltSpeed);
       CDSBeltController.setIdleMode(IdleMode.kBrake);
       SmartDashboard.putString("CDS Belt Direction", "Reverse");
-      SmartDashboard.putNumber("CDS Belt Speed", Constants.CDSBeltSpeed);
+      SmartDashboard.putNumber("CDS Belt Speed", -Constants.CDSBeltSpeed);
+
     } else {
-      CDSWheelControllerOne.getSparkMax().set(Constants.CDSWheelControllerSpeed);
-      SmartDashboard.putString("CDS Wheel Controller Direction", "Forward");
-      SmartDashboard.putNumber("CDS Wheel Controller Speed", Constants.CDSWheelControllerSpeed);
-      
-      CDSBeltController.getSparkMax().set(-Constants.CDSBeltSpeed);
+      CDSBeltController.getSparkMax().set(Constants.CDSBeltSpeed);
       SmartDashboard.putString("CDS Belt Direction", "Forward");
-      SmartDashboard.putNumber("CDS Belt Speed", -Constants.CDSBeltSpeed);
+      SmartDashboard.putNumber("CDS Belt Speed", Constants.CDSBeltSpeed);
     }
   }
 
   public void stopCDS() {
+    // stops all motors in the CDS
     CDSWheelControllerOne.getSparkMax().set(0.0);
     CDSBeltController.getSparkMax().set(0.0);
+    SmartDashboard.putNumber("CDS Wheel Speed", 0.0);
     SmartDashboard.putNumber("CDS Belt Speed", 0.0);
   }
+
+  public void stopCDSWheel() {
+    // Stops only the centering wheels
+    CDSWheelControllerOne.getSparkMax().set(0.0);
+    SmartDashboard.putNumber("CDS Wheel Speed", 0.0);
+  }
   
   public int[] getSensorStatus() {
-    int frontStatus = colorSensorOne.getProximity() > 800 ? 1: 0;
-    int backStatus = backBeamBreak.get() ? 1: 0;
-    int[] beamBreakArray = {frontStatus, backStatus};
+    int[] sensorStatuses = colorSensors.getProximities();
+
+    int frontStatus = sensorStatuses[0] > 300 ? 1: 0;
+    int middleStatus = sensorStatuses[1] > 450 ? 1: 0;
+    int backStatus = sensorStatuses[2] > 600 ? 1: 0;
+    int[] beamBreakArray = {backStatus, middleStatus, frontStatus};
```

Comment:
The ordering gets a bit confusing since beamBreakArray uses back-to-front order and sensorStatuses uses front-to-back. You can change the order sensorStatuses gets by changing how ColorSensorMuxed is constructed. i.e. 
```
 colorSensors = new ColorSensorMuxed(2,1,0);
```
Will result in getting back-to-front ordering

Or you can change which I2C port the sensors are plugged in to for the same result. (This is possibly better because then the array indices line up with the I2C Port numbers)